### PR TITLE
VTAdmin: Distributed transactions list on VTAdmin

### PR DESCRIPTION
**Pull request by beingnoble03**
## Description
- Includes `TimeCreated` while querying unresolved transactions
- Adds VTAdmin API endpoint to fetch the transactions by `keyspace`
- Initiates the Transactions screen on VTAdmin web.

Sample output from API:
```json
{
    "result": {
        "transactions": [
            {
                "dtid": "1",
                "state": 1,
                "participants": [
                    {
                        "keyspace": "commerce",
                        "shard": "0",
                        "tablet_type": 1
                    }
                ]
            },
            {
                "dtid": "2",
                "state": 1,
                "participants": [
                    {
                        "keyspace": "commerce",
                        "shard": "0",
                        "tablet_type": 1
                    }
                ]
            }
        ]
    },
    "ok": true
}
```

## Related Issue(s)
- #16766

## Screenshots
image
Screenshot 2024-09-20 at 2 10 49 AM

## Checklist

-   [ ] "Backport to:" labels have been added if this change should be back-ported to release branches
-   [ ] If this change is to be back-ported to previous releases, a justification is included in the PR description
-   [ ] Tests were added or are not required
-   [ ] Did the new or modified tests pass consistently locally and on CI?
-   [ ] Documentation was added or is not required

### CODE REVIEW

**Comment by harshit-gangal** [comment]
We should call it `In Flight Distributed Transactions` on the `Transactions` page.
Also, we can ignore the tablet type information of the shard as all will be `Primary` only.

**Comment by harshit-gangal** [comment]
We should also display the age of the transaction to know from how long the transaction is unresolved.

**Review by rohit-nayak-ps — APPROVED** [review]

File: web/vtadmin/src/util/time.ts [hunk 1]
```diff
@@ -13,7 +13,7 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-import * as dayjs from 'dayjs';
+import dayjs from 'dayjs';
```
Comment (harshit-gangal) [inline web/vtadmin/src/util/time.ts:16]: nit: I am not sure what we follow in website. Let us import all explicitly.

**Review by harshit-gangal — COMMENTED** [review]
When we call for getting unresolved transactions for a keyspace.
There is a request parameter called `abandonAge`.
We should use that and may be provide options to user to tell how old the transaction should be to display on VTAdmin.
`5s`, `30s`, `5min`, etc.

File: web/vtadmin/src/util/time.ts (hunk 1 above)
Comment (harshit-gangal) [reply web/vtadmin/src/util/time.ts:16]: that is the change here. ignore the comment

**Comment by harshit-gangal** [comment]
This can also be done in a follow up PR.

**Pull request merged**